static/compiled/application.css

```diff
@@ -0,0 +1,1171 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
These files shouldn’t be under source control—this used to be where compiled asset files lived, now moved to `dist/`